src/templates/common/mediaImage/index.tsx

```diff
@@ -31,17 +29,15 @@ export const MediaImageComponent = (props: MediaImageType) => {
   return (
     <>
       {showImage && (
-        <>
-          <Image
-            id={image.id}
-            alt={image.alt}
-            src={imagePath.href || image.url}
-            width={imagePath.width || image.width}
-            height={imagePath.height || image.height}
-            className={props?.className}
-          />
-          <span>{image.title}</span>
```

Comment:
Do we need to keep this logic?

Reply:
No, we moved that into the image component directly after finding out there was support for title and tooltips 

```        <Image
          id={image.id}
          alt={image.alt}
          src={imagePath.href}
          width={imagePath.width}
          height={imagePath.height}
          title={image.title}
          className={props?.className}
        />```